### DISCUSSION TITLE
iouring system call interface is not supported

### DISCUSSION THREAD

**Discussion by Janry** [dragonflydb/dragonfly#334] [Q&A] [answered]
**Describe the bug**
Quick start with docker compose get error
I20220927 12:45:03.816557     1 init.cc:56] dragonfly running in opt mode.
I20220927 12:45:03.816596     1 dfly_main.cc:219] Starting dragonfly df-v0.8.0-7ed808d5d093ec472de4868e02182cb1432c6967
E20220927 12:45:03.816608     1 dfly_main.cc:239] iouring system call interface is not supported. Exiting...

>   dragonfly:
>     image: 'docker.dragonflydb.io/dragonflydb/dragonfly'
>     ulimits:
>       memlock: -1
>     ports:
>       - "${DRAGONFLY_PORT:-0}:6379"
>     network_mode: host
>     command: "--cache_mode true"
>     restart: '${RESTART_POLICY:-on-failure:10}'
>     volumes:
>       - ${DRAGONFLY_DATA_LOCATION:-dragonflydata}:/data

**Environment (please complete the following information):**
 - OS: Windows 11, WSL2 - [Ubuntu 22.04 LTS]
 - Containerized?: [Docker version 20.10.17, build 100c701]
 - Dragonfly Version: [df-v0.8.0]

**Comment by romange** [comment, marked as answer]
How is wsl not having iouring interface is a bug in dragonfly? 
 
What's your kernel version?  Check here when iouring was added to wsl. 
https://learn.microsoft.com/en-us/windows/wsl/kernel-release-notes

**Reply by bigdaronlee163** [reply]
iouring is not supported in  wsl ? 
![image](https://user-images.githubusercontent.com/49524138/193489970-9e7bb6e3-6283-4439-be71-e2be9513a990.png)
 5.10.16.3-microsoft-standard-WSL2  is not equal linux  kernel 5.10

**Comment by romange** [comment]
@Janry  Our latest Dragonfly version does not require iouring. Please tell me if it solves your issue.

**Reply by Janry** [reply]
@romange I solved my problem with up linux kernel. All works. Thank you for your help.
